Sources/Apodini/Jobs/Scheduler.swift

```diff
@@ -0,0 +1,120 @@
+import Foundation
+import SwifCron
+import NIO
+
+/// A convenient interface to schedule background running tasks on an event loop using `Job`s and crontab syntax.
+///
+/// `Job`s can either be scheduled at server startup using the `Schedule` configuration or from an `Handler` by using the `@Environment` property wrapper.
+/// `Job`s can use all property wrappers that are not request based.
+///
+/// ```
+/// struct Greeter: Handler {
+///     @Environment(\.scheduler) var scheduler: Scheduler
+/// }
+/// ```
+public class Scheduler {
+    internal static var shared = Scheduler()
+
+    internal var jobConfigurations: [ObjectIdentifier: JobConfiguration] = [:]
+    
+    private init() { }
+    
+    /// Schedules a `Job` on an event loop.
+    ///
+    /// ```
+    /// enqueue(Job(), with: "* * * * *", \KeyStore.job, on: request.eventLoop
+    /// ```
+    ///
+    /// This method throws an exception if the `Job` uses request based property wrappers or the crontab cannot be parsed.
+    ///
+    /// - Parameters:
+    ///     - job: The background running task conforming to `Job`s.
+    ///     - with: Crontab as a String.
+    ///     - runs: Number of times a `Job` should run.
+    ///     - keyPath: Associates a `Job` for later retrieval.
+    ///     - on: Specifies the event loop the `Job` is exectured on.
+    public func enqueue<K: ApodiniKeys, T: Job>(_ job: T,
```

Comment:
```suggestion
    /// - Throws: If the `Job` uses request based property wrappers or the crontab cannot be parsed.
    public func enqueue<K: ApodiniKeys, T: Job>(_ job: T,
```

No? 🙂

Same for `Schedule.dequeue`. Maybe it's just a matter of taste.

Reply:
I totally agree, I think this makes it clearer